pkgs/applications/networking/cluster/terraform-providers/default.nix

```diff
@@ -87,17 +87,24 @@ let
   removed-providers =
     let
       archived = name: date: throw "the ${name} terraform provider has been archived by upstream on ${date}";
+      license = name: date: throw "the ${name} terraform provider removed from nixpkgs on ${date} because of unclear licensing";
       removed = name: date: throw "the ${name} terraform provider removed from nixpkgs on ${date}";
     in
     lib.optionalAttrs config.allowAliases {
       b2 = removed "b2" "2022/06";
+      checkly = license "checkly" "2022/11";
       checkpoint = removed "checkpoint" "2022/11";
+      cloudscale = license "cloudscale" "2022/11";
+      dhall = license "dhall" "2022/11";
```

Comment:
cc @baloo

Reply:
GitHub seems to be pedantic about the license formatting. Not sure that's the 80 columns or what.
https://github.com/awakesecurity/terraform-provider-dhall/commit/a00787e90547c6bf2e7e6ad5571d7b304a85aa1e
That fixed it.
```
$ curl -L -s "https://api.github.com/repos/awakesecurity/terraform-provider-dhall/license" | jq -r '.license.spdx_id'
BSD-3-Clause
```

I released a new 0.0.5 which should be up on hashicorp registry in a few minutes. @zowoq if you could re-run a `pkgs/applications/networking/cluster/terraform-providers/update-provider awakesecurity/dhall` that'd be appreciated.


@Mic92 thanks for the notification!